_posts/2016-03-14-public-speaking-part2-where-to-start.markdown

```diff
@@ -0,0 +1,69 @@
+
+Sidenote — if your city does not (yet) have a great meet-up, it’s well worth starting one.
+
+
+### Conferences
+
+Often considered the most intimidating of all. People have purchased tickets and possibly travelled from far. The audience is usually in the hundreds, occasionally even more than a thousand. Even the most experienced speakers can get a bit nervous for this.  
+
+Funnily enough, my first ever talk, since starting out as a developer, was at a [conference][iosdevuk]. It happened to have a very friendly atmosphere and really wasn’t that scary. I can’t stress enough that, yes, the audience wants you to succeed. I remember talking through my introduction and seeing all the curious, attentive faces. They were there because they were interested. They cared for what I had to say.
+
+I will say that this doesn’t necessarily mean you should go talk about anything your heart desires. Ultimately the attendees are there to learn, so it’s only fair that as speakers you’d tailor your presentation towards your audience. But more on that in Part 3.
+
+To be considered, often conference organisers ask people to submit a CfP (Call for Papers / Call for Proposals). This is your chance to submit details on what you’d like to present on. You usually want to include a preliminary talk title, an abstract, and a bit of background on yourself. If you’ve done any talks before and they happen to have been recorded, it’s worth including the video or otherwise previous slidedecks.
+
+As for how CfPs then get chosen depends on the conference. Most of the time the organisational committee selects talks from the proposal list. Some conferences have [an independent panel][altconf] review anonymised proposals or have a system in place where [the community votes][uikonf]. Once it’s been decided, they will get back in touch with you.
+
+Usually conferences are swarmed with CfPs so please don’t be discouraged if it takes a few tries to get picked. It’s understandable but bare in mind that a friend of mine felt like this 2 years ago also. Yet he kept at it and got invited to NSSpain 2014. He got incredibly active in the SF Swift meet-up scene and open source. All of this got the ball rolling and now he's given talks all around the world. [You might even have heard of him][twt_jp].
+
+### Lightning talks
+
+If a CfP is a bit too much to prepare at the moment (we’re all busy) or you didn’t get in, another option is to give a lightning talk if the conference of your choice has slots. When done well, they are great.
+
+A lightning talk is usually capped at 5 minutes, and gives you the time and space to cover one thing well. The best ones I’ve heard, have had one main message to convey and did so excellently.
+
+They require a bit of planning as you want to be concise, but will be done before you know it and gives the audience a great point of reference to engage with you. Not to mention that sometimes you might get asked to do a full talk based off these.
```

Comment:
They require a bit of planning as you want to be concise - could you link to something like this? http://quoteinvestigator.com/2012/04/28/shorter-letter/